plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml

```diff
@@ -12,23 +12,35 @@
   </extensionContent>
 
   <extensionContent path="tutorials/@" style="css/intro.css">
-    <group style-id="content-group">
+    <group style-id="content-group" label="Google Cloud Tools for Eclipse">
       <link id="cloud-tools-for-eclipse-tutorial-link"
-            label="Google Cloud Tools for Eclipse Tutorial"
+            label="App Engine Standard Environment"
+            style-id="content-link"
+            url="https://cloud.google.com/eclipse/docs/creating-new-webapp">
+        <text>Learn how to build, run, debug, and deploy App Engine standard environment applications with Eclipse.</text>
```

Comment:
I see you removed the period for the release notes item. Remove the period to be consistent?

Reply:
done